cmd/server-main.go

```diff
@@ -504,6 +504,8 @@ func serverMain(ctx *cli.Context) {
 		if errors.Is(err, context.Canceled) {
 			logger.FatalIf(err, "Server startup canceled upon user request")
 		}
+
+		logger.FatalIf(err, "Unable to initialize the server")
```

Comment:
No we shouldn't fail server @vadmeste this is an unexpected change.

Reply:
> No we shouldn't fail server @vadmeste this is an unexpected change.

Maybe we need a comment then, it is not clear why we shouldn't fail when initServer() returns an error.

Reply:
Because the server is operational for other reasons, data is readable and accessible. There are no real reasons why server should die. 

It is implemented on purpose to avoid taking down the server instead keep it operational for runtime fixes.